### DISCUSSION TITLE
Sideload Localization files no longer updated

### DISCUSSION THREAD

**Discussion by tonito659** [requarks/wiki#3857] [Error / Bug Report] [answered]
Hello all, 

We are currently deploying a wiki.js instance in an offline context, as such, we needed to sideload the locale to make it work. 
However, it appears that the nightly export for [https://github.com/Requarks/wiki-localization](https://github.com/Requarks/wiki-localization) is no longer working. 

As such, offline instances are missing on new translations. What would be necessary to make this export working again ?

**Comment by stuart-shelton-contino** [comment]
<img width="566" alt="Screenshot 2021-05-20 at 14 49 52" src="https://user-images.githubusercontent.com/68700726/118993525-7d597f80-b97d-11eb-8f88-31f1f7007e5a.png">

This issue means that the `convert` dialog lacks human-readable strings in the latest release when running an air-gapped instance.

**Comment by NGPixel** [comment, marked as answer]
Fixed. The lokalise cli was outdated and no longer working.

**Comment by tonito659** [comment]
Thanks for the fix, translations are working wonderfully !
